.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.0
```

Comment:
This is not enough to get it building.  The old scripts directory needs to go and this entire file needs to be replaced by a [more modern script](https://github.com/typelift/Concurrent/blob/master/.travis.yml).